deno/payloads/v8/channel.ts

```diff
@@ -1011,6 +1015,19 @@ export interface APIActionRowComponent extends APIBaseMessageComponent<Component
 	components: Exclude<APIMessageComponent, APIActionRowComponent>[];
 }
 
+export interface ModalSubmitComponent {
```

Comment:
This should go in modalSubmit, not here. By making modal action row use this interface, the types are wrong.